### PR TITLE
updates

### PR DESCRIPTION
**Pull request by tbushman**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
  
- [ ] Closes currently open issue: Closes #XXX

#### Description

Upgrades necessitated formatting changes (prettier)

### CODE REVIEW

**Comment by lasjorg** [comment]
Does ChromeDriver need to be downgraded further? Or does the Chrome version on CI need to be updated? Seeing as the CI with Firefox passes I'm guessing it's just an issue with the ChromeDriver version.


I don't really know what to do here, I don't have much experience yet with doing PR reviews and it's usually just a diff check or using Gitpod on the main fCC repo.

Here is what I did:

```
git fetch upstream pull/1149/head:upgrade

git checkout upgrade

npm i

yarn format

yarn build && yarn test
```


All tests are passing.

**Comment by tbushman** [comment]
@lasjorg yeah, I've gone the rounds on this and IDK what to do. I do think updates / upgrades are in order for this repo. It passes locally because you can control which version of Chrome you're using, but I am not sure how to specify a specific version of chrome in travis.yml

**Comment by tbushman** [comment]
This might work: https://stackoverflow.com/a/57577404/3530394

**Comment by lasjorg** [comment]
Yes I saw that [here](https://travis-ci.community/t/how-to-setup-chromedriver-74-with-chrome-74-for-travis/2678) as well and was considering it.

I was just thinking that if we can avoid using a before_script like that it would be worth testing the `DETECT_CHROMEDRIVER_VERSION` env option first.

https://www.npmjs.com/package/chromedriver#detect-chromedriver-version
https://config.travis-ci.com/ref/env

I just don't understand why travis is reporting "Your Chrome version is 62.0.3202.94", shouldn't the `addons: chrome: stable` config option give us the latest stable version and not some old version? I feel like I'm getting a 404 on the CHROMEDRIVER install because it's such an old version and if the Chrome version was newer it might work.

Looking at the docs, I can't tell if the "trusty" dist actually works with `addons: chrome: stable` or if we have to use [xenial](https://docs.travis-ci.com/user/reference/xenial/)

> [docs.travis-ci: Google Chrome](https://docs.travis-ci.com/user/chrome)
> 
> For Linux, you must be running on Ubuntu Xenial 16.04 or later build environments.


Edit: Well then, using xenial worked.
https://github.com/freeCodeCamp/testable-projects-fcc/pull/1154

**Comment by lasjorg** [comment]
I can't tell if it was just switching to the xenial that did it or if we still need the DETECT_CHROMEDRIVER_VERSION env option. I guess it can't hurt to have it.

Not sure if there are any issues with switching the dist, or how to test for it?

**Comment by tbushman** [comment]
@lasjorg Looks like the main change would be xvfb setting  
https://docs.travis-ci.com/user/trusty-to-xenial-migration-guide#3-headless-browser-testing  

Not sure if we need to change what we have. Seems to be working on your test PR.

File: .travis.yml [hunk 1]
```diff
@@ -49,14 +52,6 @@ before_install:
       sudo apt-get install -y -qq ubuntu-restricted-addons;
     fi
 before_script:
-  - |
```
Comment (tbushman) [inline .travis.yml:52]: the xvfb services key on line 9 allows removal of these xvfb settings

**Pull request merged**